ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/party_management_service.proto

```diff
@@ -54,6 +51,7 @@ service PartyManagementService {
   // - ``UNAUTHENTICATED``: if the request does not include a valid access token
   // - ``PERMISSION_DENIED``: if the claims in the token are insufficient to perform a given operation
   // - ``UNIMPLEMENTED``: if synchronous party allocation is not supported by the backing participant
+  // - ``DEADLINE_EXCEEDED``: if the request times out
```

Comment:
Going forward it seems to me we want to thing of signalling errors in terms of error categories, not gRpc status codes. If so, then at some point it could be easier, from maintenance standpoint, to list error categories here instead.